### PR TITLE
Removing robots + sitemaps

### PR DESCRIPTION
**Pull request by maisiembleton**
Removing robots + sitemaps + related! 

The robots and sitemaps will all be generated with gatsby, so no longer needed here. 

NOTE: this should be merged after the [robots.txt PR ](https://github.com/sharesight/static-www/pull/549)

[Asana ticket ](https://app.asana.com/0/1202529446124674/1202557665702211)

### CODE REVIEW

File: source/sitemapindex.xml.erb [hunk 1]
```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
```
Comment (kylorhall) [inline source/sitemapindex.xml.erb:2]: Technically this file will have no replacement.  Did Adriana pass this by Jaywing for review?

File: source/sitemap.xml.erb [hunk 2]
```diff
@@ -1,23 +0,0 @@
-<% locale_obj = locals[:locale_obj] || default_locale_obj %>
```
Comment (kylorhall) [inline source/sitemap.xml.erb:1]: Technically this file will 404.  Should we redirect this to `/gatsby/sitemapindex.xml` or something…per locale?  Did Adriana pass this by Jaywing for review?

@Billyhaine can assist with redirects–I expect they will work on a `sitemap.xml`, but I don't believe we have any redirects with "extensions".

**Review by kylorhall — APPROVED** [review]
Looks fine, but yeah, deleting stuff 🤞 ensure it's been thought through!

**Review by Billyhaine — APPROVED** [review]
🤞

File: source/sitemap.xml.erb (hunk 2 above)
Comment (maisiembleton) [reply source/sitemap.xml.erb:1]: [Response in this ticket](https://app.asana.com/0/1202529446124674/1202557665702228)

File: source/sitemapindex.xml.erb (hunk 1 above)
Comment (maisiembleton) [reply source/sitemapindex.xml.erb:2]: [Response in this ticket](https://app.asana.com/0/1202529446124674/1202557665702228)

**Pull request merged**